Add 2201.5.3 release information (#7810)
ballerina-platform/ballerina-dev-website: downloads/swan-lake-release-notes/swan-lake-2201.5.3/RELEASE_NOTE.md

---
layout: ballerina-left-nav-release-notes
title: 2201.5.3 (Swan Lake) 
permalink: /downloads/swan-lake-release-notes/2201-5-3/
active: 2201-5-3
redirect_from: 
    - /downloads/swan-lake-release-notes/2201-5-3
    - /downloads/swan-lake-release-notes/2201.5.3/
    - /downloads/swan-lake-release-notes/2201-5-3-swan-lake/
    - /downloads/swan-lake-release-notes/2201-5-3-swan-lake
---

## Overview of Ballerina Swan Lake 2201.5.3

> <em>Swan Lake 2201.5.3 is the third patch release of Ballerina 2201.5.0 (Swan Lake Update 5) and it includes a new set of bug fixes to the language.</em>

Reviewer: ```suggestion
<em>Swan Lake 2201.5.3 is the third patch release of Ballerina 2201.5.0 (Swan Lake Update 5) and it includes a new set of improvements and bug fixes to the runtime, Ballerina library, and developer tools.</em>
```

## Update Ballerina

Update your current Ballerina installation directly to 2201.5.3 using the [Ballerina Update Tool](/learn/update-tool/) as follows.

1. Run `bal update` to get the latest version of the Update Tool.
2. Run `bal dist pull 2201.5.3` to update to this latest distribution.

## Install Ballerina

If you have not installed Ballerina, then, download the [installers](/downloads/#swanlake) to install.

> ## Language updates

Reviewer: ```suggestion
## Runtime updates
```

### Bug fixes

> To view bug fixes, see the [GitHub milestone for 2201.5.3 (Swan Lake)](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.5.3+label%3AType%2FBug+is%3Aclosed).

Reviewer: ```suggestion
To view bug fixes, see the [GitHub milestone for 2201.5.3 (Swan Lake)](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.5.3+label%3AType%2FBug+is%3Aclosed).
```
This is a runtime issue.

> ## Standard library updates

Reviewer: ```suggestion
## Ballerina library updates
```
